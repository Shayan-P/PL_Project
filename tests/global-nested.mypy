a = 5;
def fx():
    a=2;
    def fy():
        global a;
        print(a);
        return a;
        ;
    fy();
    return fy;
    ;

print(fx()());
