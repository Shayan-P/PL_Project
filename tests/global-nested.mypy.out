2
5
5
'finished
